src/test/java/org/links/objecthash/ObjectHashTest.java

```diff
@@ -0,0 +1,68 @@
+package org.links.objecthash;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class ObjectHashTest {
+
+  private void runTest(String json, String expectedHash) throws Exception {
+    ObjectHash r = ObjectHash.commonJsonHash(json);
+    ObjectHash e = ObjectHash.fromHex(expectedHash);
+    assertEquals(e, r);
+  }
+
+  @Test
+  public void testEmptyList() throws Exception {
+    runTest("[]",
+        "acac86c0e609ca906f632b0e2dacccb2b77d22b0621f20ebece1a4835b93f6f0");
```

Comment:
Java is using the common_json.test file now so is in sync with the other languages.